### ISSUE TITLE
A11y_Model Builder_Model Builder_Consume_Keyboard: With keyboard "Add to solution" and "Next step" controls are not accessible.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2671]
### GitHub Tags
#WCAG2.1.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Keyboard; #Tabstops; #A11yAuto

### Environment Details
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. 
7. Complete the required details of All the three (scenario, environment, data, Train, Evaluate) steps.
8. Consume screen will appear.
9. Now tab till project section
10. Now Verify that with keyboard "Console app Add to solution", "Web API Add to solution" and "Next step" controls are not accessible.

### Actual Result
with keyboard "Console app Add to solution", "Web API Add to solution" and "Next step" controls are not accessible.
**Note:
We are facing similar issue throughout the application.**

### Expected Result
"Console app Add to solution", "Web API Add to solution" and "Next step" controls should be accessible with keyboard.

### User Impact
Screen reader users will not be able to understand the name of the control if it is not defined.

### Attachment
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/78345ee4-8749-43a2-a69b-aee81007be42)
![Bug_Keyboard](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/0094f672-1a6d-4319-b811-fcaa88e68f09)

**Comment by zewditu** [comment]
@Harika1702  , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by manaswita-chougule** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
![keyboard accessible](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/9150dab8-ce80-4a65-bf2c-d7312b09f980)
